packages/calypso-build/package.json

```diff
@@ -69,9 +70,9 @@
 		"terser-webpack-plugin": "2.2.1",
 		"thread-loader": "2.1.3",
 		"typescript": "3.7.3",
-		"webpack": "4.41.2",
 		"webpack-cli": "3.3.10",
 		"webpack-filter-warnings-plugin": "1.2.1",
-		"webpack-rtl-plugin": "2.0.0"
+		"webpack-rtl-plugin": "2.0.0",
+		"webpack": "4.41.2"
```

Comment:
This is a weird sorting. Since when is `AB` sorted after `ABCD`?

Reply:
That is weird, let me see where that went wrong…

Reply:
1ee3047 fixes